src/devices/MettlerToledo/README.md

```diff
@@ -0,0 +1,40 @@
+﻿# Mettler Toledo
+
+## Summary
+This binding provides a higher-level interface for interacting with [Mettler Toledo](https://www.mt.com)-brand scales.
+
+## Device Family
+This binding should work with any MT-SICS-compatible scale that uses MT-SICS Level 0. Some Level 1 commands are also available.
```

Comment:
Is there a way to get a list of the functions implemented and not implemented at the end of this readme at least for the level 0 and 1? 
I see 3 levels of commands. so if none from level 2 and level 3 are implemented, fine to say none and don't go into the details. That will make the life easier for anyone who wants to improve later on and add missing commands.